client/components/departments_slide.js

```diff
@@ -0,0 +1,46 @@
+import React, {Component} from 'react'
+import {Link} from 'react-router-dom'
+import {Carousel} from 'react-bootstrap'
+
+const Departments_slide = () => {
```

Comment:
wtf

Reply:
the name of the component and the file is changed.